src/sqlite-xamarin/sqlite-xamarin.mdproj

```diff
@@ -0,0 +1,60 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ItemType>GenericProject</ItemType>
+    <ProjectGuid>{B8F799C5-D7CE-4E09-9CE6-BAA4173E7EC8}</ProjectGuid>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'">
+    <OutputPath>..\..\bin\Debug\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
+    <OutputPath>..\..\bin\Release\</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <BuildDependsOn>
+        _CreateProject;
+        _NdkBuild;
+    </BuildDependsOn>
+  </PropertyGroup>
+
+  <Import Project="..\..\Configuration.props" />
+
+  <!-- Build target and co. -->
+
+  <Target Name="Build" DependsOnTargets="$(BuildDependsOn)" />
+
+  <Target Name="_CreateProject">
+    <Exec Condition="!Exists('src')"
+        Command="$(ANDROID_SDK_PATH)\tools\android create project -g -n sqlitesample -p . -a MainActivity -t android-10 --package com.xamarin.sqlitesample -v 1.0.0" />
+    <Copy Condition="!Exists('src\stamp')"
+        SourceFiles="src-jni\Application.mk;src-jni\Android.mk"
```

Comment:
Should this be in `jni`, not `src-jni`? I've never heard or seen of a `src-jni` directory before, [while `jni` seems conventional](https://developer.android.com/ndk/guides/android_mk.html):

> The `Android.mk` file resides in a subdirectory of your project's `jni/` directory

That's also what `src/monodroid` uses: `src/monodroid/jni`.